src/markov/markov_approx.jl

```diff
@@ -130,3 +130,50 @@ function _rouwenhorst(p::Real, q::Real, m::Real, Δ::Real, n::Integer)
         return linspace(m-Δ, m+Δ, n), θN
     end
 end
+
+
+"""
+Accepts the simulation of a discrete state Markov chain and estimates
+the transition probabilities via maximum likelihood.
```

Comment:
There's a nice reference here:

http://www.stat.cmu.edu/~cshalizi/462/lectures/06/markov-mle.pdf

It's a nice and often cited result that max likelihood estimation of MC transition probabilities reduces to setting p_{ij} equal to the empirical frequency of transitions from i to j in the data.  This reference is also good:

https://stats.stackexchange.com/questions/47685/calculating-log-likelihood-for-given-mle-markov-chains